### PR TITLE
Feature/prepare for opam repo

### PR DESCRIPTION
**Pull request by mikonieminen**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+build:
+	@dune build @install
+
+clean:
+	@dune clean
+
+test-all:
+	SIHL_ENV=test dune runtest --force --no-buffer test
```
Comment (joseferben) [inline Makefile]: @mikonieminen We can remove `SIHL_ENV` imo

File: README.md [hunk 2]
```diff
@@ -45,3 +45,9 @@ And finally review that the email is correctly generated in the service:
 # Credits
 
 This project is build on [colombe](https://github.com/mirage/colombe "colombe project") and [mrmime](https://github.com/mirage/mrmime "mrmime project") libraries and use [facteur](https://github.com/dinosaure/facteur "facteur email sending tool") as starting point.
+
+## License
+
+Copyright (c) 2020 Miko Nieminen
```
Comment (jerben) [inline README.md:51]: @mikonieminen Should we put the same claim as in Sihl "Oxidizing Systems"?

File: Makefile (hunk 1 above)
Comment (mikonieminen) [reply Makefile]: oops

Comment (mikonieminen) [reply Makefile]: fixed

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: ocaml/opam2:4.08
+        environment:
```
Comment (joseferben) [inline .circleci/config.yml:6]: @mikonieminen This looks wrong

Comment (mikonieminen) [reply .circleci/config.yml:6]: It's copied from sihl

Comment (joseferben) [reply .circleci/config.yml:6]: Hmm I think you are right

**Comment by joseferben** [comment]
Approved.

**Pull request merged**